### PR TITLE
[ISSUE #19981] adding notifications to 'Publish CDK Manually'

### PR DESCRIPTION
**Pull request by maxi297**
## What
Adding notifications on release

### CODE REVIEW

**Comment by clnoll** [comment]
@maxi297 do you have screenshots of the notifications that you could add to the PR description?

**Comment by maxi297** [comment]
> @maxi297 do you have screenshots of the notifications that you could add to the PR description?

@clnoll I did a test on `#issue-19981_test-release-notification` that looks like this:
image

File: .github/workflows/publish-cdk-command-manually.yml [hunk 1]
```diff
@@ -58,6 +58,34 @@ jobs:
...
           commit_user_email: octavia-squidington-iii@users.noreply.github.com
+      - name: Post failure to Slack channel dev-connectors-extensibility
+        if: !success()
+        uses: slackapi/slack-github-action@v1.23.0
+        continue-on-error: true
+        with:
+          channel-id: C02TL38U5L7
+          payload: |
+            {
+                "text": "Error while publishing Airbyte CDK!"
```
Comment (clnoll) [inline .github/workflows/publish-cdk-command-manually.yml:69]: Any reason not to give a more specific message depending on whether the error occurred during `bump-version`, `build-cdk`, or `publish-cdk`?

**Review by clnoll — CHANGES_REQUESTED** [review]
Looks good, just 1 q for you @maxi297.

File: .github/workflows/publish-cdk-command-manually.yml (hunk 1 above)
Comment (maxi297) [reply .github/workflows/publish-cdk-command-manually.yml:69]: We can. I was checking the process and I think that in any case, the user will have to click on the link of the workflow to have more detail and therefore, he will have the information anyway. If we think it has value to be added earlier (like in the Slack message), let me know and I can add it

Comment (clnoll) [reply .github/workflows/publish-cdk-command-manually.yml:69]: I think that extra bit of detail would be nice to have.

File: .github/workflows/publish-cdk-command-manually.yml [hunk 2]
```diff
@@ -58,6 +58,34 @@ jobs:
           commit_message: 🤖 Bump ${{ github.event.inputs.release-type }} version of Airbyte CDK
           commit_user_name: Octavia Squidington III
           commit_user_email: octavia-squidington-iii@users.noreply.github.com
+      - name: Post failure to Slack channel dev-connectors-extensibility
```
Comment (brianjlai) [inline .github/workflows/publish-cdk-command-manually.yml:61]: My main worry about posting this to our main channel is that we might start to spam or clutter it if we have a lot of deploys or failed workflows in any given day. How would you feel about a side channel #airbyte-cdk-releases or something like that?

Comment (brianjlai) [reply .github/workflows/publish-cdk-command-manually.yml:61]: sounds good thank you. if alerts ultimately are not too noisy i have no concerns moving it to main channel.

**Review by clnoll — APPROVED** [review]
LGTM!

**Pull request merged**